src/main/java/com/volasoftware/tinder/utils/BotInitializer.java

```diff
@@ -46,17 +47,45 @@ public class BotInitializer {
 
     private final NameGenerator nameGenerator;
 
+    private final JdbcTemplate jdbcTemplate;
+
     @PostConstruct
     void initialize() {
-        Page<Account> botAccounts = accountRepository.findByAccountType(
-            AccountType.BOT,
-            PageRequest.of(0, 10)
-        );
+        setupBots();
+    }
+
+    private void buildInitialAccounts() {
+        Account realAccount1 = createAccount("Ivan", "Tsenov", "ivan.t.tsenov@gmail.com",
+            "IVNtsn963", Gender.MALE, Role.USER, false, true, 18, null,
+            AccountType.REAL);
+        Account realAccount2 = createAccount("Toni", "Tsenova", "toni.k.tsenova@gmail.com",
+            "TNSLVtsn963", Gender.FEMALE, Role.USER, false, true, 18, null,
+            AccountType.REAL);
+
+        accountRepository.save(realAccount1);
+        accountRepository.save(realAccount2);
+    }
```

Comment:
Why we want to create these two users?

Reply:
I created 2 initial users in relation with TIN-15, where I must link real users with bots, so I can test linking requested Real account with bots or linking all accounts, when the id is not provided by calling the endpoint.

Reply:
I see, remove that logic, please!